README.md

```diff
@@ -86,31 +97,46 @@ Make sure this is consistent with the redirect URI used in `okta_app_auth_config
 if your **Redirect URI** is `com.okta.example:/callback`, the **AppAuth Redirect Scheme** should be
 `com.okta.example`.
 
+#### Chrome Custom Tabs `ERR_UNKNOWN_URL_SCHEME`
+
+There is a [known issue](https://github.com/okta/okta-sdk-appauth-android/issues/8) when redirecting back to the a URI scheme from the browser via Chrome Custom Tabs. This is due to Chrome **not supporting** JavaScript initiated redirects back to native applications.
+
+To handle this, the lead maintainer of AppAuth proposes the following:
```

Comment:
Not sure if we need to callout "lead maintainer" - would be enough to just say the AppAuth team